MAINTAINERS.md

```diff
@@ -131,13 +131,13 @@ An automated list can be found [here](https://github.com/orgs/tldr-pages/people)
 - Axel Navarro ([@navarroaxel](https://github.com/navarroaxel)):
   [5 October 2020](https://github.com/tldr-pages/tldr/issues/4504) — [7 April 2021](https://github.com/tldr-pages/tldr/issues/5703)
 - bl-ue ([@bl-ue](https://github.com/bl-ue)):
-  [2 February 2021](https://github.com/tldr-pages/tldr/issues/5219) — [25 June 2021](https://gitter.im/tldr-pages/tldr?at=60d615598a40b117282a96d7)
+  [2 February 2021](https://github.com/tldr-pages/tldr/issues/5219) — [25 June 2021](https://matrix.to/#/!zXiOpjSkFTvtMpsenJ:gitter.im/$qCyBANu8Ub_GKJgwh0zKlVSgWASLYxYJXBn4NDEEQPw?via=gitter.im&via=matrix.org&via=one.ems.host)
```

Comment:
The previous link seems to go straight to the message, whereas the new one shows me the "You're invited to talk on Matrix" dialog. I'm not sure it's a net benefit to make this change :thinking:

Reply:
(Not to mention it a waaaay uglier URL :stuck_out_tongue_winking_eye:)

Reply:
> (Not to mention it's a waaaay uglier URL 😜)

Unfortunately, irrespective of the matrix client the same URL/permalink will be generated.


> The previous link seems to go straight to the message, whereas the new one shows me the "You're invited to talk on Matrix" dialog. I'm not sure it's a net benefit to make this change 🤔

That one is annoying but the old one won't work anymore, so this is the only alternative. Also, once you have selected the client, you don't need to login to see the message.

Reply:
I see, I clicked the old URL and for some reason was under the impression that it redirected me to the right place. That's really a bummer, but I hope their upcoming public archive view may improve the situation.

For the time being, though, I wonder if we need the `?via=gitter.im&via=matrix.org&via=one.ems.host` part of the URL. I believe those can be dropped, right?

Reply:
> For the time being, though, I wonder if we need the `?via=gitter.im&via=matrix.org&via=one.ems.host` part of the URL. I believe those can be dropped, right?

I will test it out tomorrow, It's late here. If it works will update it.

Reply:
Nicer link: https://matrix.to/#/#tldr-pages:matrix.org/$qCyBANu8Ub_GKJgwh0zKlVSgWASLYxYJXBn4NDEEQPw

We should drop the query params as Waldy suggests. (Imo it's just a good habit to strip them off before sharing links, even personally.)

Reply:
> Nicer link: https://matrix.to/#/#tldr-pages:matrix.org/$qCyBANu8Ub_GKJgwh0zKlVSgWASLYxYJXBn4NDEEQPw
> 
> We should drop the query params as Waldy suggests. (Imo it's just a good habit to strip them off before sharing links, even personally.)

Yay, seems to work will update it now before going to sleep lol.

Reply:
Done, Updated the query params

Reply:
Cool, thanks both. I'll mark this thread as resolved.